functionary/core/utils/listener.py

```diff
@@ -0,0 +1,55 @@
+import json
```

Comment:
This whole file is again duplicative of the runner listener code.  Both this and the runner code will be refactored at some point so that they share a singular listener / worker code-base.